src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5339,4 +5339,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_OutVariableCannotBeByRef" xml:space="preserve">
     <value>An out variable cannot be declared as a ref local</value>
   </data>
+  <data name="CanCreateConstructedFromConstructed" xml:space="preserve">
+    <value>Can't create constructed generic type from another constructed generic type.</value>
+  </data>
+  <data name="CantCreateConstructedFromNongeneric" xml:space="preserve">
+    <value>Can't create constructed generic type from non-generic type.</value>
```

Comment:
while we are not 100% consistent here, the general loc feedback is that contractions should be avoided.  So "cannot create".  This also matches messages like "cannot create static class".

Reply:
@CyrusNajmabadi The one message I saw that starts with "Can" does use a contraction:

https://github.com/dotnet/roslyn/blob/ddd03771a64f10cbc3abdff4bcb8ad3438667770/src/Compilers/CSharp/Portable/CSharpResources.resx#L4314-L4316

Contractions are also [recommended by the Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions).

That's just to explain why I wrote it that way initially, I have changed it now.